core/src/main/java/org/opensearch/sql/data/model/ExprTimeValue.java

```diff
@@ -51,6 +56,21 @@ public LocalTime timeValue() {
     return time;
   }
 
+  @Override
+  public LocalDate dateValue() {
+    return LocalDate.now();
+  }
+
+  @Override
+  public LocalDateTime datetimeValue() {
+    return LocalDateTime.of(dateValue(), timeValue());
+  }
+
+  @Override
+  public Instant timestampValue() {
+    return ZonedDateTime.of(dateValue(), timeValue(), ZoneId.of("UTC")).toInstant();
```

Comment:
Is the intent for `timestampValue` to return the same value as if it was converted to `ExprTimestampValue`?

If so, it'd be best to use `ExprTimestampValue.ZONE`. That way if timestamp ever supports non-UTC timezones, it will be clear this line needs to be updated as well.

Reply:
Fixed in 8cd8f783, thanks.